src/generated/resources/openapi.yaml

```diff
@@ -1188,13 +1188,13 @@ components:
     HealthCheck:
       type: object
       properties:
-        status:
-          enum:
-          - UP
-          - DOWN
+        name:
           type: string
         data:
           type: object
           nullable: true
-        name:
+        status:
+          enum:
+          - UP
+          - DOWN
```

Comment:
Unrelated to this PR. No idea why this shuffles around

Reply:
The OpenAPI spec generator does not deterministically generate the health check or response types, and will alternate as other changes are made in the codebase.

I would just manually exclude those line changes from the PR. I often will add line changes manually for the `openapi.yml` and `openapi.json` files, excluding any health-related changes (unless that's intentional), rather than just adding all changes in those files.